examples/reference/elements/bokeh/Tiles.ipynb

```diff
@@ -54,6 +54,13 @@
    "source": [
     "One thing to note about tile sources is that they are always defined in the [pseudo-Mercator projection](https://epsg.io/3857), which means that if you want to overlay any data on top of a tile source the values have to be expressed as eastings and northings. If you have data in another projection, e.g. latitudes and longitudes, it may make sense to use [GeoViews](http://geoviews.org/) for it to handle the projections for you.\n",
     "\n",
+    ">\n",
+    "> **Note**: \n",
+    "> \n",
+    ">   Holoviews provides functions to project longitude, latitude into Web Mercator coordinates. \n",
+    ">   See [`hv.util.transform.lon_lat_to_easting_northing(longitude, latitude)`](https://holoviews.org/reference_manual/holoviews.element.html?highlight=lon_lat_to_easting_northing#holoviews.element.Tiles.lon_lat_to_easting_northing)\n",
+    ">\n",
+    "\n",
```

Comment:
Document holoviews function used to transform coordinates from (long, lat) to web mercator.